engine/apps/slack/alert_group_slack_service.py

```diff
@@ -61,22 +56,8 @@ def update_alert_group_slack_message(self, alert_group: "AlertGroup") -> None:
                 raise e
 
         except SlackAPIException as e:
-            if e.response["error"] == "message_not_found":
-                logger.info(f"message_not_found for alert_group {alert_group.pk}, trying to post new message")
-                result = self._slack_client.api_call(
-                    "chat.postMessage", channel=slack_message.channel_id, attachments=attachments, blocks=blocks
-                )
-                slack_message_updated = SlackMessage(
-                    slack_id=result["ts"],
-                    organization=slack_message.organization,
-                    _slack_team_identity=slack_message.slack_team_identity,
-                    channel_id=slack_message.channel_id,
-                    alert_group=alert_group,
-                )
-                slack_message_updated.save()
-                alert_group.slack_message = slack_message_updated
-                alert_group.save(update_fields=["slack_message"])
-                logger.info(f"Message has been posted for alert_group {alert_group.pk}")
```

Comment:
Removing this logic completely because it's hard to maintain with the new FK structure.
I think it makes more sense to just ignore deleted AG Slack messages instead of trying to post a new message (why post a message one more time if the user already decided they don't want to see it and deleted it?)